tests/test_meshing_queries.py

```diff
@@ -0,0 +1,123 @@
+from ansys.fluent.core import examples
+
+import_filename = examples.download_file(
+    "elbow.cas.h5", "pyfluent/examples/DOE-ML-Mixing-Elbow"
+)
+
+
+def test_meshing_queries(new_mesh_session):
+    meshing_session = new_mesh_session
+    meshing_session.tui.file.read_case(import_filename)
+
+    assert meshing_session.meshing_queries.GetFaceZoneAtLocation(
+        [1.4, 1.4, 1.4]
+    ) == meshing_session.scheme_eval.scheme_eval(
+        "(get-face-zone-at-location '(1.4 1.4 1.4))"
+    )
+
+    assert meshing_session.meshing_queries.GetZonesOfType(
+        "velocity-inlet"
+    ) == meshing_session.scheme_eval.scheme_eval("(get-zones-of-type 'velocity-inlet)")
+
+    assert meshing_session.meshing_queries.GetZonesOfGroup(
+        "inlet"
+    ) == meshing_session.scheme_eval.scheme_eval("(get-zones-of-group 'inlet)")
+
+    assert meshing_session.meshing_queries.GetFaceZonesOfFilter(
+        "*"
+    ) == meshing_session.scheme_eval.scheme_eval("(get-face-zones-of-filter '*)")
+
+    assert meshing_session.meshing_queries.GetCellZonesOfFilter("*") == [3460]
+    assert meshing_session.scheme_eval.scheme_eval("(get-cell-zones-of-filter '*)") == (
+        3460,
+    )
+
+    assert meshing_session.meshing_queries.GetEdgeZonesOfFilter("*") == []
+    assert (
+        meshing_session.scheme_eval.scheme_eval("(get-edge-zones-of-filter '*)") == None
+    )
```

Comment:
same is applicable here.

Reply:
In scheme_eval serve-side, scheme nil `()` is converted to empty gRPC message which is converted to `None` in Python. This is correct behaviour as scheme_eval is a generic interface. Whereas `GetEdgeZonesOfFilter` should always return a list.

I think what you've done here is very clear and fine.